Gemfile.tt

```diff
@@ -4,35 +4,54 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 ruby File.read(".ruby-version")
 
 gem "rails", "<%= Rails.version %>"
-gem "puma", "~> 4.1"
-gem "pg"<%= gemfile_requirement("pg") || ', "~> 0.18"' %>
+gem "puma"
+gem "pg"
```

Comment:
constraining versions seems like the wrong move in a template like this. I can't think of a situation where we wouldn't want to default to the latest version of almost all gems. If there exceptions we need to lock then we can do so but the risk is that they stay locked here for a long time e.g. `rack-canonical-host` (see below) is up to version 1.1 now